src/polydb.jl

```diff
@@ -181,39 +185,53 @@ function _get_contact(a::Array)
 end
 
 # returns information String about a specific section
-function _get_section_string(db::Database, name::String)
+function _get_section_string(db::Database, name::String, level::Int64)
    info = _get_info_document(db, string("_sectionInfo.", name))
-   res = [string("SECTION: ", join(info["section"], "."), "\n", info["description"])]
-   if haskey(info, "maintainer")
+   res = [string("SECTION: ", join(info["section"], "."))]
+   if level == 1 && haskey(info, "short_description")
+      push!(res, string("\t", info["short_description"]))
+   end
+   if level >= 2 && haskey(info, "description")
+      push!(res, info["description"])
+   end
+   if level >= 3 && haskey(info, "maintainer")
       push!(res, string("Maintained by ", info["maintainer"]["name"], ", ", info["maintainer"]["email"], ", ", info["maintainer"]["affiliation"]))
    end
    return join(res, "\n")
 end
 
 # returns information String about a specific collection
-function _get_collection_string(db::Database, name::String)
+function _get_collection_string(db::Database, name::String, level::Int64)
    info = _get_info_document(db, string("_collectionInfo.", name))
    res = [string("\tCOLLECTION: ", name)]
-   if haskey(info, "description")
+   if level == 1 && haskey(info, "short_description")
+      push!(res, string("\t", info["short_description"]))
+   end
+   if level >= 2 && haskey(info, "description")
       push!(res, string("\t", info["description"]))
    end
-   if haskey(info, "author")
+   if level >= 3 && haskey(info, "author")
       push!(res, string("\tAuthored by ", "\n", _get_contact(info["author"])))
    end
-   if haskey(info, "maintainer")
+   if level >= 3 && haskey(info, "maintainer")
       push!(res, string("\tMaintained by", "\n", _get_contact(info["maintainer"])))
    end
+   if level >= 5
+      push!(res, string("\tFields: ", _get_field_string(db[name])))
+   end
    return join(res, "\n")
 end
 
 # prints a sorted list of the sections and collections of the Polydb
 # together with information about each of these, if existent
 # relying on the structure of Polydb
-function info(db::Database)
+function info(db::Database, level::Int64)
```

Comment:
given that on L:233 we define `info(db) = info(db,1)` we could just define
```suggestion
function info(db::Database, level=1)
```
Is `Int64` particularly needed? As far as I can see it's only used for comparison. In such case we can even omit the signature, or if You want to strictly type turn it to `Integer`